gen/definitions/profile_parcels/system_snmp.yaml

```diff
@@ -0,0 +1,101 @@
+---
+name: System SNMP
+rest_endpoint: /v1/feature-profile/sdwan/system/%v/snmp
+minimum_version: 20.12.0
+attributes:
+  - tf_name: feature_profile_id
+    reference: true
+    type: String
+    description: Feature Profile ID
+    example: f6dd22c8-0b4f-496c-9a0b-6813d1f8b8ac
+    test_value: sdwan_system_feature_profile.test.id
+  - model_name: shutdown
+    example: false
+  - model_name: contact
+    tf_name: contact_person
+    example: "wixie.cisco"
+  - model_name: location
+    tf_name: location_of_device
+    example: "SHANGHAI"
+  - model_name: view
+    tf_name: views
+    attributes:
+      - model_name: name
+        id: true
+        example: VIEW1
+      - model_name: oid
+        attributes:
+          - model_name: id
+            id: true
+            example: 1.3.6.1.4.1.9.9.394
+          - model_name: exclude
+            example: false
+  - model_name: community
+    tf_name: communities
+    attributes:
+      - model_name: name
+        write_only: true
+        example: example
+      - model_name: userLabel
+        id: true
+        example: COMMUNITY1
+      - model_name: view
+        example: VIEW1
+      - model_name: authorization
+        example: read-only
+  - model_name: group
+    tf_name: groups
+    attributes:
+      - model_name: name
+        id: true
+        example: GROUP1
+      - model_name: securityLevel
+        example: "auth-priv"
+      - model_name: view
+        example: VIEW1
+  - model_name: user
+    tf_name: users
+    attributes:
+      - model_name: name
+        id: true
+        example: USER1
+      - model_name: auth
+        tf_name: authentication_protocol
+        example: "sha"
+      - model_name: authPassword
+        tf_name: authentication_password
+        example: "$CRYPT_CLUSTER$su56l1Z0Tk4Qc9N7+T/uOg==$sD6b0HLqEdI+RNwsEOoLcQ=="
+      - model_name: priv
+        tf_name: privacy_protocol
+        example: "aes-256-cfb-128"
+      - model_name: privPassword
+        tf_name: privacy_password
+        example: "$CRYPT_CLUSTER$su56l1Z0Tk4Qc9N7+T/uOg==$sD6b0HLqEdI+RNwsEOoLcQ=="
+      - model_name: group
+        example: GROUP1
+  - model_name: target
+    tf_name: trap_target_servers
+    attributes:
+      - model_name: vpnId
+        tf_name: vpn_to_reach_trap_target_server
```

Comment:
i think `vpn_id` should be fine here.. the parent element is already named `trap_target_servers`